uui/components/pickers/PickerToggler.tsx

```diff
@@ -27,44 +24,14 @@ function applyPickerTogglerMods(mods: PickerTogglerMods) {
     ];
 }
 
-function PickerTogglerComponent<TItem extends string, TId>(props: PickerTogglerProps<TItem, TId> & PickerTogglerMods, ref: React.ForwardedRef<HTMLElement>) {
-    const getPickerTogglerButtonSize = (propSize: types.ControlSize) => {
-        switch (propSize) {
-            case '48':
-                return '42';
-            case '42':
-                return '36';
-            case '36':
-                return '30';
-            case '30':
-                return '24';
-            case '24':
-                return '18';
-        }
-    };
-
-    const getCaption = (row: DataRowProps<TItem, TId>) => {
-        const maxItems = getMaxItems(props.maxItems);
-
-        if (row.isLoading) {
-            return <TextPlaceholder />;
-        } else if (!props.getName || props.selectedRowsCount > maxItems) {
-            return row.value;
-        } else {
-            return props.getName(row.value);
-        }
-    };
-
-    const renderItem = (row: DataRowProps<TItem, TId>) => (
-        <Tag
-            key={ row.rowKey }
-            caption={ getCaption(row) }
-            tabIndex={ -1 }
-            size={ props.size ? getPickerTogglerButtonSize(props.size) : '30' }
-            onClear={ () => {
-                row.onCheck?.(row);
-            } }
-            isDisabled={ props.isDisabled || props.isReadonly || row?.checkbox?.isDisabled }
+function PickerTogglerComponent<TItem extends string, TId>(props: PickerTogglerProps<TItem, TId> & PickerTogglerMods, ref: React.ForwardedRef<HTMLElement>): JSX.Element {
+    const renderItem = (itemProps: PickerTogglerTagProps<TItem, TId>) => (
+        <PickerTogglerTag
+            { ...itemProps }
+            rowProps={ itemProps.rowProps }
+            key={ itemProps.isCollapsed ? 'collapsed' : itemProps.rowProps?.id as string }
+            size={ props.size }
+            isDisabled={ props.isDisabled || props.isReadonly || itemProps.isDisabled }
```

Comment:
I supposed that we shouldn't make any props calculation here, we should receive final prop from params and here just spread it to the component, i believe that default renderItem callback should look like:
   
```
   const renderItem = (itemProps: PickerTogglerTagProps<TItem, TId>) => (
        <PickerTogglerTag { ...itemProps } /> 
        )
```

Reply:
I left only size here, or do we want to move it to the uui-components?